modules/nodalsAiRtdProvider.js

```diff
@@ -246,11 +246,11 @@ class NodalsAiRtdProvider {
    */
 
   #hasRequiredUserConsent(userConsent) {
-    if (userConsent?.gdpr?.gdprApplies !== true) {
+    if (userConsent?.gdpr?.gdprApplies === false) {
       return true;
     }
     if (
-      userConsent?.gdpr?.vendorData?.vendor?.consents?.[this.gvlid] === false
+      [false, undefined].includes(userConsent?.gdpr?.vendorData?.vendor?.consents?.[this.gvlid])
     ) {
       return false;
```

Comment:
if the `consentManagementTcf` module is not present `userConsent.gdpr` would be null and you'd end up in the second `if` - which I think would prevent your module from running. Is that the intent - restrict it only to GDPR publishers?

Reply:
one option to consider is something like https://github.com/prebid/Prebid.js/blob/c6dc75b76d5ff02a42c0251eb4c994cdba03a8f1/libraries/smartyadsUtils/getAdUrlByRegion.js#L7 to not worry about the tcf module being null if the user is not in europe, but require that it is non-null there

Reply:
@dgirardi / @patmmccann  - We're happy to go with:
 - If gdpr status is unknown: let the server make the decision based on the data subject's geo
 - if gdpr status is known: allow if flagged outside of jurisdiction. If within jurisdiction, check vendor consent, purpose 1 and purpose 7